### PR TITLE
Adding link to cron doc.

### PR DESCRIPTION
**Pull request by Ernestopheles**
may be helpful, f.i. good to know you should add a HTTP request context.

### CODE REVIEW

File: docs/manual/performance/cronjobs.de.md [hunk 1]
```diff
@@ -30,3 +30,7 @@ Beispiel könnte so aussehen:
 ```
 * * * * * /opt/plesk/php/8.2/bin/php /var/www/vhosts/my.host.com/vendor/bin/contao-console contao:cron
 ```
+
+{{% notice tip %}}
+See the [Contao Cron Documentation](https://docs.contao.org/dev/framework/cron/)
```
Comment (netzarbeiter) [inline docs/manual/performance/cronjobs.de.md]:
```suggestion
Zusätzliche Informationen findest du in [der englischen Entwickler-Dokumentation](https://docs.contao.org/dev/framework/cron/).
```

File: docs/manual/performance/cronjobs.de.md [hunk 2]
```diff
@@ -30,3 +30,7 @@ Beispiel könnte so aussehen:
...
+for more details.
```
Comment (netzarbeiter) [inline docs/manual/performance/cronjobs.de.md]:
```suggestion
{{% /notice %}}
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Ernst.

**Review by netzarbeiter — APPROVED** [review]

**Comment by Ernestopheles** [comment]
Thankx

**Pull request merged**